riscv-asm.md

```diff
@@ -954,7 +954,7 @@ jr rs                        | jalr x0, rs, 0
 jalr rs                      | jalr x1, rs, 0                                                | Jump and link register
 ret                          | jalr x0, x1, 0                                                | Return from subroutine
 call offset                  | auipc x1, offset[31:12]; jalr x1, x1, offset[11:0]            | Call far-away subroutine
-tail offset                  | auipc x6, offset[31:12]; jalr x0, x6, offset[11:0]            | Tail call far-away subroutine
+tail offset                  | auipc x6, offset[31:12]; jalr x0, x6, offset[11:0]            | Tail call far-away subroutine | It will use `x7` as scratch register when `Zicfilp` extension enabled.
```

Comment:
"when `Zicfilp` extension enabled" -> "when the `Zicfilp` extension is available"